### PR TITLE
Trace filters get additional parameter with method/sub name

### PR DESCRIPTION
**Pull request by roonyh**

### CODE REVIEW

**Comment by arunoda** [comment]
This looks good. Let's extend our two existing filters to write like this.

```
Kadira.tracer.addFilter(Kadira.Tracer.stripSensitive([ "start", "http", "email" ], 'method', 'abc'));
```

File: lib/tracer/default_filters.js [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+    strippedTypes[type] = true;
   });
 
-  return function (type, data) {
-    if(typesToStrip.length > 0 && !allowedTypes[type]) return data;
+  return function (type, data, info) {
+    if(typesToStrip.length > 0 && !strippedTypes[type])
+      return data;
+
+    if(receiverType && receiverType!=info.type)
```
Comment (arunoda) [inline lib/tracer/default_filters.js]: Add spaces before and after `!=`

File: lib/tracer/default_filters.js [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
+      return data;
+
+    if(name && name!=info.name)
```
Comment (arunoda) [inline lib/tracer/default_filters.js]: Add spaces before and after `!=`

**Comment by arunoda** [comment]
This is pretty solid. Add this to `stripSelectors` default filter as well.

**Comment by roonyh** [comment]
@arunoda, Added to `stripSelectors`

**Comment by arunoda** [comment]
Did you add(or update) test cases for those changes?

**Comment by roonyh** [comment]
Yes. Added [here](https://github.com/meteorhacks/kadira/pull/195/files#diff-0b95e269700aab90e3cfd83da8ce25b4R123)

**Comment by arunoda** [comment]
Okay great. I'll take these in and do a release.

**Pull request merged**